rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+- All links on a page have a [link equity](https://moz.com/learn/seo/what-is-link-equity) and our own internal links are competing for this equity. We need to add nofollow to any link that we think should not earn a share. Attention: we can be penalized for linking to bad quality content but also for linking too much to some content! We should dive deeper into this and see if this means we should add nofollow to most of our AWS docs links.
+- We could be more intentional about our robots.txt. Not crawling irrelevant pages or low performing pages raises the average performance of the website, which raises the ranking of the website.
+- Landing pages might not be a good idea after all. Thin content used to be good SEO practice (have multiple pages for each thing), but this does not have lots of advantages to users so now Google penalizes this. As a rule of thumb, what's good for users will be good for rankings since Google uses machine learning on user behavior. So we should watch things like time on page, bounce rate etc. identify low performing pages vs high performing pages and perhaps remove or improve low performing pages.
+
+![https://moz.imgix.net/learn/guides/Moz-BGSEO-Ch4-01-outlines-01.svg?auto=format&ch=Width&fit=max&q=50&s=e236ac8928a1cd1940243f15ce7dbb6d](https://moz.imgix.net/learn/guides/Moz-BGSEO-Ch4-01-outlines-01.svg?auto=format&ch=Width&fit=max&q=50&s=e236ac8928a1cd1940243f15ce7dbb6d)
+
+- According to the poing above, user engagement becomes one of the most important things on a website. Which means adding the comment section is a good idea.
+
+- Keywords are important but sometimes the most authoritative keywords are not the best ones. High-volume keywords are also more competitive to rank higher (keyword difficulty). Carefully [exploring the long tail](https://moz.com/blog/long-tail-seo-target-low-volume-keywords-whiteboard-friday) of relevant keywords to us might offer us a better advantage in ranking and in financial conversions.
+
+  > For example, a person searching for "shoes" is probably just browsing. On the other hand, someone searching for "best price red womens size 7 running shoe" practically has their wallet out!
+
+- Perhaps we were not intentional enough concerning keywords in our current content. According to the point above, phrases formulated as questions are SEO gold. We could do a search on the most common questions on our space and make sure these questions are phrased in our content
+
+- Writing content is HARD so we need to make sure we are making the most for our buck. We need a strategy before deciding which content to write about. I've seen these two content strategies which seem very interesting
```

Comment:
I think our #1 driver for what content we write should be guides that customers are asking for. E.g., we had _many_ customers get stuck on the basic account + IAM setup. And we'll have many more who need help with K8S services, secrets management, CI / CD, and a handful of other topics. So, even if it might not perform as well SEO-wise, I still want that to be our #1 driver.

However, once we've settled on the content we want to write, the other ideas in here seem like an excellent way to get the most bang for the buck from that content. Also, once we have the core guides in place and start to approach the long tail, searching for "10x content" is also a great idea.

Reply:
> I think our #1 driver for what content we write should be guides that customers are asking 

I agree 100%! My intention on bringing this up is for the case where we have more than one request or possibility and this could help us pick which one we write first and ways to tweak it to rank higher

Reply:
Roger